doc/examples/transform/plot_transform_types.py

```diff
@@ -0,0 +1,146 @@
+# For a rotation around the center of the image, one can 
+# compose a translation to change the origin, a rotation, and finally
+# the inverse of the first translation.
+# in degrees, while it is given in radians for the different kinds of
+# affine transformations):
+
+rotation = tf.EuclideanTransform(rotation=np.pi/3)
+shift = tf.EuclideanTransform(translation=-np.array(img.shape[:2]) / 2)
+# Compose transforms by multiplying their matrices
+matrix = np.linalg.inv(shift.params) @ rotation.params @ shift.params
+tform = tf.EuclideanTransform(matrix)
+tf_img = tf.warp(img, tform.inverse)
+fig, ax = plt.subplots()
+ax.imshow(tf_img)
+
+######################################################################
+# Similarity transformation
+# =================================
+#
+# A `similarity transformation <https://en.wikipedia.org/wiki/Similarity_(geometry)>`_
+# preserves the shape of objects. It combines scaling, translation and rotation.
+
+tform = tf.SimilarityTransform(
+    scale=0.5,
+    rotation=np.pi/12,
+    translation=(100, 50))
+print(tform.params)
+tf_img = tf.warp(img, tform.inverse)
+fig, ax = plt.subplots()
+ax.imshow(tf_img)
+ax.set_title('Similarity transformation')
+
+######################################################################
+# Affine transformation
+# =================================
+#
+# An `affine transformation <https://en.wikipedia.org/wiki/Affine_transformation>`_
+# preserves lines (hence the alignment of objects), as well as parallelism
+# between lines. It can be decomposed into a similarity transform and a
+# `shear transformation <https://en.wikipedia.org/wiki/Shear_mapping>`_.
+
+tform = tf.AffineTransform(
+        shear=np.pi/6,
+        )
+print(tform.params)
+tf_img = tf.warp(img, tform.inverse)
+fig, ax = plt.subplots()
+ax.imshow(tf_img)
+ax.set_title('Affine transformation')
+
+
+######################################################################
+# Projective transformation (homographies)
+# ========================================
+#
+# A `homography <https://en.wikipedia.org/wiki/Homography>`_, also called
+# projective transformation, preserves lines but not necessarily 
+# parallelism.
+
+matrix = np.array([[1, -0.5, 100],
+                   [0.1, 0.9, 50],
+                   [0.0015, 0.0015, 1]])
+tform = tf.ProjectiveTransform(matrix=matrix)
+tf_img = tf.warp(img, tform.inverse)
+fig, ax = plt.subplots()
+ax.imshow(tf_img)
+ax.set_title('Projective transformation')
+
+plt.show()
+
+
+
+######################################################################
+# See also
+# ========================================
+#
+# * :ref:`sphx_glr_auto_examples_transform_plot_geometric.py` for composing
+#   transformations or estimating their parameters
+# * :ref:`sphx_glr_auto_examples_transform_plot_rescale.py` for simple
+#   rescaling and resizing operations
+# * :func:`skimage.transform.rotate` for rotating an image around its center
+
+# sphinx_gallery_thumbnail_number = 5
```

Comment:
This shows up in the html page I generated. I guess there is a little issue. Perhaps adding a # sign above solves this.

Reply:
I found a sphinx-gallery option to strip those :-).

Reply:
great!